app/retail/templates/home/index2020.html

```diff
@@ -220,45 +220,20 @@ <h2>Mission</h2>
 
     <div class="href hover-logo" href="/results">
       <div>
-        <h1>73k</h1>
+        <h1>90k</h1>
         <h2>Developers</h2>
       </div>
     </div>
 
     <div class="href hover-logo" href="/results">
       <div>
-        <h1>$13.5mm</h1>
+        <h1>$20mm</h1>
```

Comment:
why's this $20mm and not $20m ?

Reply:
I'm not sure , but do check the https://github.com/gitcoinco/web/issues/8695
I assumed it was a typo but it's been like that all this while 😅